src/ui/components/shared/IconWithTooltip.tsx

```diff
@@ -16,16 +18,15 @@ export default function IconWithTooltip({
   handleClick,
 }: IconWithTooltipProps) {
   const { onMouseEnter, onMouseLeave, tooltip } = useTooltip({
-    className: "icon-tooltip",
-    delay: 250,
+    delay: 120,
     position: "right-of",
     tooltip: content,
   });
 
   return (
-    <div className="icon-with-tooltip text-sm">
+    <div>
       <button
-        className="icon-with-tooltip-button"
+        className={styles.iconWithTooltipButton}
```

Comment:
Can you move the style (`styles.iconWithTooltipButton`) to the icon, since it is what you're trying to style anyway? The `IconWithTooltip` component is only used in on place (`Header`) so you should be able to just do it there:

https://github.com/replayio/devtools/blob/4ec18e26da579cba99808a4877195ab9c3f5a277/src/ui/components/Header/Header.tsx#L169